doc/source/ray-core/tasks/fault-tolerance.rst

```diff
@@ -8,7 +8,10 @@ because the process crashed or because the machine failed, Ray will rerun
 the task until either the task succeeds or the maximum number of retries is
 exceeded. The default number of retries is 3 and can be overridden by
 specifying ``max_retries`` in the ``@ray.remote`` decorator. Specifying -1
-allows infinite retries, and 0 disables retries.
+allows infinite retries, and 0 disables retries. To override the default number
+of retries for all tasks submitted, set the OS environment variable
+``RAY_task_max_retries``. e.g., by passing this to your driver script or by
+using :ref:`runtime environments<runtime-environments>`.
```

Comment:
By convention, I think we should add an alias so this is `RAY_TASK_MAX_RETRIES`, and reserve `RAY_lower_case_vars` as internal flags, for user clarity.

I'll leave it to you and @scv119 to decide if this should be a policy though.

Reply:
Makes sense, I'll change this.